Prep DB for rolling back the Access DB migration

johnclary opened the pull request:
This PR addresses the need to be able to cleanly rollback the Access DB migration.

- Add a  field will allow us totrack any projects created through the Access DB migration so that we can delete them if needed. We can delete this column after we successfully migrate. 
- Disable Hasura event triggers on `delete` operations. We do not allow records to be deleted through Hasura except by our admin/root user. This change removes Hasura `delete` event triggers entirely, because the only conditions under which we are deleting records will be for admin purposes that we presumably to not want to track as events.

In order to completely roll back the migration we'll need to run [these commands](https://github.com/cityofaustin/atd-moped/blob/4838-jc-migrate-access-db/moped-toolbox/access_migration/migrate_data/rollback.sql). I intend to package that up as a single executable.

Steps to test
1. Just start your local DB
2. Use a SQL make sure that deleting project data does not trigger events:

```sql
-- clear all record from the event invocation lgos
truncate hdb_catalog.event_invocation_logs;

-- delete all projects;
delete from moped_project where 1=1;

-- verify no events were invoked from the deletion
select * from hdb_catalog.event_invocation_logs;
```

mddilley reviewed (approved):
Spun up locally and the migrations check out! 🚀

frankhereford reviewed (approved):
Starts right up! Code looks great - thank you John.

frankhereford commented:
@chiaberry pointed out to me that you had removed your request for reviews! Sorry for jumping the gun. I thought you had invited the dev group and it had removed the rest of us when Mike reviewed. My bad - looking forward to this in final form.

Thanks John!

johnclary commented:
Thanks for the quick reviews Frank and Mike! I just added another change to this PR, updated the test steps, and requested review again. Would y'all mind looking at this again at your convenience?

mddilley reviewed (approved):
Tested locally, and I see that record deletes no longer invoke activity log event triggers. 🚀 🚢

chiaberry reviewed (approved):
same! test steps check out

johnclary commented:
Thank you!
